<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>debug_console</class>
 <widget class="QDialog" name="debug_console">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit_SerialInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>440</width>
     <height>350</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_OptionsBaudRate">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>110</y>
     <width>85</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_OptionsNBits">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>150</y>
     <width>85</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>5 bits</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6 bits</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7 bits</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8 bits</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_OptionsParity">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>190</y>
     <width>85</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>No Parity</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Even Parity</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Odd Parity</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_OptionsPortName">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>85</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_PortName">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Port Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_BaudRate">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Baud rate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_NBits">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>N-Bits</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Parity">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>63</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Parity:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Connection">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>290</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Pause">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ClearInput">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>330</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Save">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>370</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SendCMD">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>430</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Send CMD</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_SerialCommand">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>440</width>
     <height>30</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Type your command&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_DEBUGModeConsole">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;DEBUG Mode Console&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_StopBit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Bit:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_OpetionsStopBit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>230</y>
     <width>85</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1 bit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2 bits</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.5 bit</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_ClearCMD">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>430</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>CLR CMD</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
